src/index.ts

```diff
@@ -91,7 +92,7 @@ export default function create<TState extends State>(
   const useStore = <StateSlice>(
     selector: StateSelector<TState, StateSlice> = getState,
     equalityFn: EqualityChecker<StateSlice> = Object.is
-  ): StateSlice => {
+  ): ReturnState<TState, StateSlice> => {
```

Comment:
The `: {TYPE}` can be removed to let TypeScript infer the return type.

Reply:
Done